apps/contrib/templates/meinberlin_contrib/item_confirm_delete.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% load i18n item_tags %}
+
+{% block title %}{% trans 'Confirm Idea Deletion' %}{% endblock %}
+
+{% block content %}
+<div class="l-wrapper resource-header">
+    <h1 class="resource-header__title">{% trans 'Are you sure you want to delete the idea?' %}</h1>
```

Comment:
The resource header component does not work here because "Are you sure you want to delete the idea?" is not the title of a resource.